src/nock/reporter.ts

```diff
@@ -0,0 +1,108 @@
+import { NonPrimitive } from '@superfaceai/one-sdk/dist/internal/interpreter/variables';
+import createDebug from 'debug';
+import { join as joinPath } from 'path';
+
+import { exists, readFileQuiet, readFilesInDir, rimraf } from '../common/io';
+import { OutputStream } from '../common/output-stream';
+import {
+  TestAnalysis,
+  TestCoverageBase,
+  TestingReturn,
+} from '../superface-test.interfaces';
+import { AnalysisResult } from './analyzer';
+
+const DEFAULT_COVERAGE_PATH = 'coverage';
```

Comment:
question: won't it be conflicting with actual test coverage?

Reply:
Good question, I'll investigate